microsoft/testsuites/cdrom/cdrom.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import pathlib
+
+from assertpy import assert_that
+
+from lisa import Node, SkippedException, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import Linux
+from lisa.sut_orchestrator import AZURE
+from lisa.testsuite import simple_requirement
+from lisa.tools import Gcc
+
+
+@TestSuiteMetadata(
+    area="cdrom",
+    category="functional",
+    description="""
+        Tests to check the behavior of the virtual cdrom device in VMs.
+    """,
+)
+class CdromSuite(TestSuite):
+    # CD drive status constants from linux uapi
+    # see https://github.com/torvalds/linux/blob/master/include/uapi/linux/cdrom.h#L417
+    _expected_device_status = (
+        "CDS_NO_DISC"  # drive tray is closed, no disk in the drive
+    )
+
+    @TestCaseMetadata(
+        description="""
+            Test to check the installation ISO is unloaded
+            after provisioning and rebooting a new VM.
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            supported_os=[Linux], supported_platform_type=[AZURE]
```

Comment:
It doesn't need to specify Linux, it's default settings.

Reply:
Does supported_os=[Linux] not enforce that other OS's are unsupported?
I've only tested on Linux and don't see the need to support BSD. It was a host side fix so whichever OS checks the status of the drive should get the same result.